src/commands/owner/aggregates/owner.aggregate.ts

```diff
@@ -91,13 +91,13 @@ export class OwnerAggregate extends AggregateRoot {
 
 interface OwnerState {
   id: string;
-  nodeIds: Array<[string, string]>;
-  ssoIds: Array<[string, string]>;
-  emails: Array<[string, string]>;
-  publicNames: Array<[string, string]>;
-  names: Array<[string, string]>;
-  companyNames: Array<[string, string]>;
-  websites: Array<[string, string]>;
+  nodeIds: Array<string>;
```

Comment:
Waarom is dit een `Array` eigenlijk? (dat zie ik nu pas 😬 )

Reply:
Dat zijn alle nodeIds vanuit alle events, de laatste is de huidige nodeId. Dus als je nodeId ophaalt wordt de functie get nodeId aangesproken, die de laatste uit deze array pakt. Op die manier werd het gedaan in de tutorial, weet niet zeker of dat de juiste manier is.